packages/client/src/views/RegisterForm/RegisterForm.tsx

```diff
@@ -686,6 +686,10 @@ class RegisterFormView extends React.Component<FullProps, State> {
         informant = 'father'
       }
 
+      // informant needs to be reset as the relationship changed
+      // see https://github.com/opencrvs/opencrvs-core/issues/5866
+      delete declaration.data.informant._fhirIDPatient
+
```

Comment:
We should not directly update the declaration data as it's in redux. Perhaps we can do something like this:
```
const { _fhirIDPatient, ...informantWithoutID } = declaration.data.informant
this.props.modifyDeclaration({
  ...declaration,
  data: {
    ...
    ...
    informant: {
      ...informantWithoutID
    }
  }
```

Reply:
what would be its advantage?

Reply:
in this case I guess it doesn't give us anything as the "_fhirIDPatient" isn't being used anywhere in the client but the reasoning behind this approach would be the "immutability pattern". Using "delete" doesn't create a new object so the redux store wouldn't have any idea that the object changed